okio-testing/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+
+kotlin {
+  jvm {
+    withJava()
```

Comment:
What Java code is in this module?

Reply:
None. I’m attempting to configure this as close as possible to the main `okio` artifact so I can later merge the two without much friction.